test/suites/beta/drift/suite_test.go

```diff
@@ -89,7 +89,9 @@ var _ = Describe("Beta/Drift", Label("AWS"), func() {
 			Spec: corev1beta1.NodePoolSpec{
 				Template: corev1beta1.NodeClaimTemplate{
 					Spec: corev1beta1.NodeClaimSpec{
-						Requirements: []v1.NodeSelectorRequirement{{Key: corev1beta1.CapacityTypeLabelKey, Operator: v1.NodeSelectorOpIn, Values: []string{corev1beta1.CapacityTypeOnDemand}}},
+						Requirements: []v1.NodeSelectorRequirement{
+							{Key: corev1beta1.CapacityTypeLabelKey, Operator: v1.NodeSelectorOpIn, Values: []string{corev1beta1.CapacityTypeOnDemand}},
+							{Key: v1.LabelArchStable, Operator: v1.NodeSelectorOpIn, Values: []string{corev1beta1.ArchitectureAmd64, corev1beta1.ArchitectureArm64}}},
```

Comment:
Supporting both architectures is implicit with beta when you don't specify the requirement so I'm not sure that you need to specify it explicitly here